session.go

```diff
@@ -383,3 +356,18 @@ func (s *Session) waitForStateOrContext(ctx context.Context, state State) {
 		s.cond.Wait()
 	}
 }
+
+// captureStdout captures the first line of stdout.  Screen emits errors to
+// stdout so this allows logging extra context beyond the exit code.
+func captureStdout(process Process) chan string {
```

Comment:
```suggestion
func captureStdout(process Process) <-chan string {
```

Reply:
Does this mark it as receive-only?  I like that.

Reply:
yep, exactly.